### PR TITLE
Remove tests for asymptotic special-cases for tan()

### PR DESCRIPTION
**Pull request by CanadaHonk**
See resolved CSS spec issue: https://github.com/w3c/csswg-drafts/issues/8527

### CODE REVIEW

**Review by annevk — COMMENTED** [review]
Hmm, where are we testing these inputs now? (Filed https://github.com/w3c/csswg-drafts/issues/8865.)

**Comment by tabatkins** [comment]
We're not; we explicitly undefined those values so they *can't* be tested.

File: css/css-values/acos-asin-atan-atan2-serialize.html [hunk 1]
```diff
@@ -43,9 +43,6 @@
     "atan(0.5)"                                         :"calc(26.5651deg)",
     "atan(0.577350269)"                                 :"calc(30deg)",
     "atan(0)"                                           :"calc(0deg)",
-    "atan(infinity)"                                    :"calc(90deg)",
```
Comment (tabatkins) [inline css/css-values/acos-asin-atan-atan2-serialize.html:46]: We didn't undefine this; it still needs to return 90deg.

**Comment by CanadaHonk** [comment]
To check, are this PR's test removals acceptable then?

**Review by tabatkins — CHANGES_REQUESTED** [review]

**Comment by tabatkins** [comment]
r+ after the one nit I just posted.

File: css/css-values/acos-asin-atan-atan2-serialize.html (hunk 1 above)
Comment (CanadaHonk) [reply css/css-values/acos-asin-atan-atan2-serialize.html:46]: Good point, fixed, thanks!

**Review by tabatkins — APPROVED** [review]

**Pull request merged**